lua/nvim-dap-virtual-text/virtual_text.lua

```diff
@@ -123,14 +123,16 @@ function M.set_virtual_text(stackframe, options)
     local query = get_query(ltree:lang(), 'locals')
     if query then
       for _, match, _ in query:iter_matches(tree:root(), buf, 0, -1) do
```

Comment:
With
```
, { all = true }
```
it would also be compatible with the old Neovim version that had `all = false` as a default.

Reply:
thanks, done